src/test/suite/telemetry/connectionTelemetry.test.ts

```diff
@@ -1,5 +1,5 @@
-import { beforeEach, afterEach } from 'mocha';
-import { connect } from 'mongodb-data-service';
+import { before, after, beforeEach, afterEach } from 'mocha';
+import { connect, DataServiceImpl } from 'mongodb-data-service';
```

Comment:
This is interesting. My branch suddenly started failing on this test suite. I copied this refactored version to it and now tests pass. Hope you don't mind I copied it here: https://github.com/mongodb-js/vscode/pull/482/commits/8d77ef332f38ebbe1aa934e9b10c337083850364#diff-a468a59574d19865f8eea5715e9ceaa187fb55debcddb46eee85c55236408c6d

Reply:
I just pushed up a new commit that removes usage of `DataServiceImpl` instead using only the `DataService` type. I think you'll want to update it again - sorry!

Reply:
It actually started to fail again with the old version I copied. Something weird is happening with this test suit.

<img width="731" alt="Screenshot 2023-02-27 at 18 09 33" src="https://user-images.githubusercontent.com/16307679/221631908-81f15778-e453-44ba-a2c4-be55f33efe72.png">